src/server/zset_family.cc

```diff
@@ -2445,6 +2468,39 @@ void ZSetFamily::GeoHash(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ZSetFamily::GeoPos(CmdArgList args, ConnectionContext* cntx) {
+  string_view key = ArgS(args, 0);
+
+  absl::InlinedVector<string_view, 8> members(args.size() - 1);
+  for (size_t i = 1; i < args.size(); ++i) {
+    members[i - 1] = ArgS(args, i);
+  }
+
+  auto cb = [&](Transaction* t, EngineShard* shard) {
+    return OpMScore(t->GetOpArgs(shard), key, members);
+  };
+
+  OpResult<MScoreResponse> result = cntx->transaction->ScheduleSingleHopT(std::move(cb));
```

Comment:
It looks like the first part of this method returns 3 times now. Maybe we can extract it to a dedicated function instead of copying it?

Reply:
makes sense. extracting it to `ZGetMembers`